Rakefile

```diff
@@ -27,7 +27,7 @@ require 'rake'
 require 'rake/clean'
 require 'rake/testtask'
 require 'rake/packagetask'
-require 'rake/contrib/rubyforgepublisher'
+# require 'rake/contrib/rubyforgepublisher'
```

Comment:
In order to run the tests in the build file I had to comment this line out.  Is there a better way to handle this?  @mwgalloway